dev/start-anvil-with-state

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Do we need this at all? Can't we use the default anvil entrypoint?

Reply:
This was here before, I am out of my depth on why this line was here to begin with.